temba/flows/models.py

```diff
@@ -5312,6 +5338,9 @@ def execute(self, run, context, actionset_uuid, msg, offline_on=None):
             if self.msg:
                 text = run.flow.get_localized_text(self.msg, run.contact)
 
+            quick_replies = ReplyAction.get_translated_metadata(self.quick_replies, run) if self.quick_replies else []
+            metadata = dict(quick_replies=quick_replies)
+
```

Comment:
Since we'll just be passing quick_replies down below, don't need to construct metadata here.